### DISCUSSION TITLE
Access to some localizations

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#21449] [Q&A] [answered]
How can I access strings from sub-categories of `common`? For example, this one:
![изображение](https://user-images.githubusercontent.com/53872073/204433756-ac8b351c-d87f-4148-b32e-e88384a41b97.png)
I tried but did not find existing usages lazer-side.

**Comment by smoogipoo** [comment, marked as answer]
`CommonStrings.ButtonsDelete` from the `osu.Game.Resources.Localisation.Web` namespace.
